### PR TITLE
📝 docs: prevent test example from breaking TS

### PR DESCRIPTION
**Pull request by Moshyfawn**
ref: https://github.com/react-hook-form/react-hook-form/discussions/7125#discussioncomment-2180265

### CODE REVIEW

File: src/data/en/advanced.tsx [hunk 1]
```diff
@@ -554,7 +554,7 @@ export default {
         </p>
         <CodeArea
           rawData={CodeExampleTestingForm.actWarningSolution}
-          url="https://codesandbox.io/s/react-hook-form-unit-test-act-warning-docs-yq7uj?file=/src/App.test.js"
+          url="https://codesandbox.io/s/react-hook-form-unit-test-act-warning-docs-mpu0o?file=/src/App.test.js"
```
Comment (bluebill1049) [inline src/data/en/advanced.tsx:557]: i have invited you to the hook form codesandbox 👍

Comment (Moshyfawn) [reply src/data/en/advanced.tsx:557]: Yep, accepted, thx! Still, can seem to update the CSB without forking it, tho

Comment (bluebill1049) [reply src/data/en/advanced.tsx:557]: interesting... there is no owner on this CSB. shall we use this one: https://codesandbox.io/s/react-hook-form-unit-test-act-warning-docs-tcb7y?

Comment (Moshyfawn) [reply src/data/en/advanced.tsx]: Updated to use the suggested CSB 👍

**Comment by bluebill1049** [comment]
thanks very much @Moshyfawn (as always)

**Pull request merged**